hubblestack/extmods/returners/sqlite.py

```diff
@@ -4,21 +4,24 @@
 """
 from __future__ import absolute_import
 
-import salt.returners
+import json
 import logging
 import os
-import json
+import salt.returners
+
 try:
     import sqlite3
+
     GOT_SQLI = True
-except:
```

Comment:
I'm surprised except Exception passes where except: doesn't... I wonder if we should select the exception for "no such module" instead.

Reply:
on second thought... who cares. this module is only for testing anyway, and if we ever use it again, we can fix it then.